jest.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
+  env: {
+    jest: true,
+  },
+  extends: ['plugin:jest/recommended', './base.js'],
```

Comment:
Yes but when you do:

extends: ['algolia/react', 'algolia/jest']

It seems you need to repeat prettier at the end, so it's always added at the last plugin.
To be fair I don't even know if that's necessary to do so. If there's a plugin order.

If not and if only the rules order is important (would make more sense). Then we could remove all those prettier duplicates in other confs.